packages/functions/src/checks/utils.ts

```diff
@@ -0,0 +1,68 @@
+import admin from "firebase-admin";
+
+import {
+  Collection,
+  OrgSubCollection,
+  SlotAttendnace,
+  SlotInterface,
+  map,
+  wrapIter,
+} from "@eisbuk/shared";
+
+type Firestore = admin.firestore.Firestore;
+
+export const findSlotAttendanceMismatches = async (
+  db: Firestore,
+  organization: string
+) => {
+  const orgRef = db.collection(Collection.Organizations).doc(organization);
+
+  const slots = await orgRef
+    .collection(OrgSubCollection.Slots)
+    .get()
+    .then((snap) => snap.docs);
+  const attendances = await orgRef
+    .collection(OrgSubCollection.Attendance)
+    .get()
+    .then((snap) => snap.docs);
+
```

Comment:
Shouldn't we also go through `OrgSubCollection.Bookings`? I'm 99% sure this work is already planned, but I mention it anyway, in case we're in that 1% case.